src/routes/NewUserPage.tsx

```diff
@@ -1,14 +1,15 @@
 import React, { FunctionComponent, useCallback, useEffect, useState, useRef } from 'react';
```

Comment:
I had initially thought that we could follow the examples of frameworks like [Next.js](https://nextjs.org/docs/routing/introduction) and [Remix](https://remix.run/docs/en/v1/guides/routing) by having everything in `/routes` mimic the path used (for example `/search.html` => `/routes/search.jsx`)... the paths we use are just terrible though, so maybe that is not the best idea for now...

I would like to hear some additional feedback on this though. :slightly_smiling_face:

Reply:
We could just overhaul the paths while we're at it

Reply:
That is another option (and one that I wanted to do at some point), but we would probably need to maintain some redirects for the old paths for people with bookmarks (or just have an open tab during an upgrade)

Reply:
we could notify to update their bookmarks on release.